css-view-transitions-1/Overview.bs

```diff
@@ -1135,9 +1143,14 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 						Note: This happens if |transition| was [=skip the view transition|skipped=] before this point.
 
+					1. [=Mark as handled=] |transition|'s [=ViewTransition/ready promise=].
```

Comment:
Likewise here, it seems that this is only a duplicate if update callback done promise is not explicitly handled? Or am I misunderstanding?